spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/support/DefaultPublisherFactory.java

```diff
@@ -61,6 +61,8 @@
 	private BatchingSettings batchingSettings;
 
 	/**
+	 * Default {@link DefaultPublisherFactory} constructor.
```

Comment:
This is somehow really non-informative JavaDoc.

We develop Open Source API and that would be great to respect end-users and let them to know what is this about. 
Just several words in the JavaDocs and a couple of your minutes and there is nothing to nit pick in the perfect Spring code!

I don't say that we should polish brilliantly our code right now, but that is exactly what we should work out in time.

Reply:
Not sure what action you'd like to see here.

Should I just remove this Javadoc bit entirely?